cloud/pulsar-admin/README.md

```diff
@@ -4,19 +4,49 @@ The `pulsar-admin` is a CLI tool written in Java language for the Apache Pulsar
 
 # Prerequisites
 
-- Pulsar broker 2.7.0-742fc5c9b+
+> Pulsar client is required to be newer than 2.6.1 which will include the OAuth2 authentication plugin.
 
-> You can get this tarball from [bintray](https://bintray.com/streamnative/maven/org.apache.pulsar/2.7.0-742fc5c9b). When Pulsar 2.6.1 is released, you can also use the official 2.6.1 version.
+- Pulsar broker 2.7.0-742fc5c9b+
+- Get the `WEB_SERVICE_URL` of your StreamNative Cloud Pulsar cluster: [How to get service URL](../README.md#How to get service URL)
+- Get the `AUTH_PARAMS` of your StreamNative Cloud Pulsar cluster: [How to get token options](../README.md#How to get token options)
```

Comment:
Is the format of the link right? 
https://github.com/streamnative/pulsar-examples/blob/c83e2818bdb1a51cf72558fbc52d54fc2537cc55/cloud/pulsar-admin/README.md
![image](https://user-images.githubusercontent.com/6348208/89702991-31eb5d00-d979-11ea-9ff5-1c476671a245.png)